AnkiDroid/src/main/java/com/ichi2/anki/cardviewer/ViewerCommand.java

```diff
@@ -72,7 +72,8 @@
     COMMAND_ABORT_AND_SYNC(R.string.gesture_abort_sync, 34),
     COMMAND_RECORD_VOICE(R.string.record_voice, 35),
     COMMAND_REPLAY_VOICE(R.string.replay_voice, 36),
-    COMMAND_TOGGLE_WHITEBOARD(R.string.gesture_toggle_whiteboard, 37);
+    COMMAND_TOGGLE_WHITEBOARD(R.string.gesture_toggle_whiteboard, 37),
+    COMMAND_SHOW_HINT(R.string.toggle_hint,38);
```

Comment:
our code style standard should enforce this? We put space between parameters https://github.com/ankidroid/Anki-Android/wiki/Code-style

```suggestion
    COMMAND_SHOW_HINT(R.string.toggle_hint, 38);
```